src/PetriParse/PNMLParser.cpp

```diff
@@ -596,7 +596,7 @@ void PNMLParser::parsePlace(rapidxml::xml_node<>* element) {
     std::string id(element->first_attribute("id")->value());
 
     auto initial = element->first_attribute("initialMarking");
-    long long initialMarking = 0;
+    uint32_t initialMarking = 0;
```

Comment:
uint64_t would be more appropriate here, and with a check that it is less than std::numeric_limits<uint32_t>::max()

Reply:
Fixed in fc39d12a8a389fe0f113b38daa2f0ef9bd386487